battleship.py

```diff
@@ -56,7 +53,7 @@ def check_availability(target, grid):
 
 
 def check_hit(target, ships):
-    if target in ships:
+    if target in ships["carrier"] or target in ships["cruiser"]:
```

Comment:
This works for now. However, when we get more ships it's going to get ugly. I know there's a more elegant way to do this with a for loop.